lib/rdkafka/bindings.rb

```diff
@@ -157,7 +171,7 @@ class TopicPartitionList < FFI::Struct
       :void, [:pointer, :int, :string, :pointer]
     ) do |_client_prr, err_code, reason, _opaque|
       if Rdkafka::Config.error_callback
-        error = Rdkafka::RdkafkaError.new(err_code, broker_message: reason)
+        error = Rdkafka::RdkafkaError.build(err_code, broker_message: reason)
```

Comment:
We are within the `Rdkafka` scope here, so we could do `RdkafkaError.build` as in the other files above.

Reply:
This should be done in a separate PR, not to cloud this one with the transaction changes.